graphql/resolvers.ts

```diff
@@ -13,6 +13,9 @@ export default {
         order: [['order', 'ASC']]
       })
     },
+    submissions() {
+      return Submission.findAll()
```

Comment:
this will be insanely slow. Our database has over 10000 submissions so far, some of them are really bad (includes huge 20 file changes). Could you accomplish what you want to do, without pulling every submission from the database?